types/evidence.go

```diff
@@ -917,6 +917,138 @@ func (e PotentialAmnesiaEvidence) String() string {
 	return fmt.Sprintf("PotentialAmnesiaEvidence{VoteA: %v, VoteB: %v}", e.VoteA, e.VoteB)
 }
 
+type ProofOfLockChange struct {
+	Votes  []Vote        `json:"votes"`
+	PubKey crypto.PubKey `json:"pubkey"`
+}
+
+var _ Evidence = &ProofOfLockChange{}
```

Comment:
does this have to adhere to Evidence interface?

Reply:
can't evidence reactor accept it as a separate message?

Reply:
The evidence reactor is bypassed - do you want me to still use the evidence reactor?

Reply:
> does this have to adhere to Evidence interface?

Yes